### PR TITLE
Registration endpoint #6

### PR DESCRIPTION
**Pull request by captbilard**
Created registration endpoint using dj-rest-auth

### CODE REVIEW

File: accounts/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.1.1 on 2020-09-01 23:36
+# Generated by Django 3.0 on 2020-09-11 15:16
```
Comment (Damephena) [inline accounts/migrations/0001_initial.py:1]: Which version of Django are you using? Did you install the requirements.txt file?

File: accounts/serializers.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from rest_framework import serializers
+
+
+from dj_rest_auth.registration.serializers import RegisterSerializer
+
+from .models import User
+
+
```
Comment (Damephena) [inline accounts/serializers.py]: pep8 spacing: Hi, it would be great to have consistent spacing in the file.

File: accounts/views.py [hunk 3]
```diff
@@ -1,3 +1,26 @@
+# Core Django imports
 from django.shortcuts import render
+from django.utils.translation import ugettext_lazy as _
+
+#Third-party app imports
+from rest_framework import status
+from rest_framework.response import Response
+from dj_rest_auth.registration.views import VerifyEmailView
+
+
```
Comment (Damephena) [inline accounts/views.py]: pep8 spacing

File: accounts/tests.py [hunk 4]
```diff
@@ -2,9 +2,14 @@
 '''
 from django.test import TestCase
 
+
+
 import accounts.models as models
```
Comment (Damephena) [inline accounts/tests.py]: pep8 spacing

**Review by Damephena — CHANGES_REQUESTED** [review]
Kindly make changes. Thanks

File: accounts/migrations/0001_initial.py (hunk 1 above)
Comment (captbilard) [reply accounts/migrations/0001_initial.py:1]: Yea I installed all required packages using the requirements.txt file.

File: accounts/tests.py (hunk 4 above)
Comment (captbilard) [reply accounts/tests.py]: Done, Kindly review.

File: accounts/migrations/0001_initial.py (hunk 1 above)
Comment (Damephena) [reply accounts/migrations/0001_initial.py:1]: Okay.

**Review by Damephena — APPROVED** [review]
👍🏽

**Pull request merged**